Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -341,8 +346,8 @@ extension MapView: DelegatingMapClientDelegate {
 
         metalView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         metalView.autoResizeDrawable = true
-        metalView.contentScaleFactor = UIScreen.main.scale
-        metalView.contentMode = .center
+        metalView.contentScaleFactor = pixelRatio
+        metalView.contentMode = .scaleToFill
```

Comment:
I think contentMode has to remain `.center`. What would be the case where content has to be scaled, especially non-proportionally?

Reply:
Without changing the content mode, it actually does not consistently fix the issue. `.scaleToFile` guarantees the fix

Reply:
Thanks, but that doesn't answer my question. What is the case where non-proportional scaling of rendered content is required?

Reply:
I see what you are saying, but I think when using `.scaleToFill` that accounts for that and properly scales the proportions. Especially when we are providing a scale factor

Reply:
Why would we want to stretch view? Could you explain why `.scaleToFill` is used instead of, for example `.scaleAspectFit`or `.scaleAspectFill` that preserve aspect ratio?

Reply:
The issues with the other two is that `.scaleAspectFit` will not fill the screen and `.scaleAspectFill` will result in some clipping. Which in theory could be accounted for by zooming.

Reply:
How could map not fit the screen? We are not rendering static content, like an image. When view is laid-out, map could always render content preserving aspect ratio.

Reply:
I mean in my testing, they all produced similar results and you do bring up a good point. I'm good with moving it to scaleAspectFill

Reply:
@alexshalamov and I discussed this and went over some examples. Keeping content mode to `.center` and specifically passing the right seale factor in fixes the issue